packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -22,6 +22,21 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
 
   private _classNames: IClassNames<IGroupHeaderStyles>;
 
+  public static getDerivedStateFromProps(newProps: IGroupHeaderProps, state: IGroupHeaderState): IGroupHeaderState {
+    if (newProps.group) {
+      const { group, isGroupLoading } = newProps;
+      const { isCollapsed = false } = group;
+      const isLoadingVisible = Boolean(!isCollapsed && isGroupLoading && isGroupLoading(group));
```

Comment:
```suggestion
      const isLoadingVisible =!isCollapsed && !!isGroupLoading && isGroupLoading(group);
```

Reply:
I had this originally but opted for `Boolean`. What is your deciding factor for boolean coercion vs. constructor?

Reply:
It might mainly be preference (unless the minifier is unable to convert `Boolean` to `!!`). Using `!!` feels more "javascript-y" whereas `Boolean` reminds me of antiquated code from people new to the language and/or forced to use that syntax by an inordinately picky internal-only type checker.

Reply:
Minification optimization is a valid reason, thanks.